spanish/wg-diversity-inclusion/focus-areas/project-and-community/README.md

```diff
@@ -1,19 +1,15 @@
-## Project & Communities
+##Proyecto y Comunidades
 
-**Goal:** Identify how diverse and inclusive our project places, where community engagement occurs, are.
+**Meta:** Identificar cuán diversos e inclusivos son los espacios de nuestro proyecto, en los lugares donde ocurren las mayores interacciones. 
 
-**Description:** 
+**Descripción:** 
 
-Name | Question
+Nombre | Pregunta
 --- | ---
-[Documentation Usability](documentation-usability.md) | What is the usability of documentation from content and structure perspectives?
-[Documentation Accessibility](documentation-accessibility.md) | How does the documentation accommodate different users?
-[Issue Label Inclusivity](issue-label-inclusivity.md) | How well are project issues labeled to invite new contributors, skilled contributors, non-code contributors, and other types of contributors?
-[Project Burnout](project-burnout.md) | How is project burnout identified and managed within an open source project?
-[Chat Platform Inclusivity](chat-platform-inclusivity.md ) | How do you review Chat Platform inclusivity for your community?
-| (The following metrics are not released, yet)
-[Communication Channels](channels.md) | How welcoming, responsive, respectful are interactions even on hot topics of debate?  What is the diversity of voices speaking/being heard?
-[Response Times & Quality](response-time-quality.md) | How quickly and well do we respond to suggestions, PRS, questions (Note: dependent on quality of PR, questions needs factored in)?
-[Sentiment](sentiment.md) | What is the sentiment within external communication channels regarding our own press releases and within our internal communication channels, e.g., mail lists or IRC?
-
-> This README needs to be translated to Spanish
\ No newline at end of file
+[Usabilidad de la Documentación](documentation-usability.md) | ¿Cuál es la usabilidad de la documentación desde el punto de vista del contenido y la estructura?
+[Accesibilidad de la Documentación](documentation-accessibility.md) | ¿Hasta qué grado se acomoda la documentación a diferentes usuarios? 
+[Descubrimiento de la Documentación](documentation_discoverability.md) | Las personas usuarias y contribuyentes, ¿pueden acceder a la documentación del proyecto y encontrar lo que buscan con facilidad?
```

Comment:
Contribuyente sounds as tax payer. Does it make sense to use "contribuidor"? => "Las personas usuarias y contribuidoras. We may use "participantes" as well.

Reply:
This discussion about "contribuyente", "contribuidor" or "participante" applies to the rest of the doc.

Reply:
I agree that "contribuyente" is commonly associated with taxpayers. In this case, I believe "colaboradores" could make a stronger case. What do you think?

Reply:
Hey guys! Thanks for checking it out. "Contribuyente" is the way the spanish github version defines contributors, the same goes for " comentarios" and "propuestas". These are terms I got from github in spanish and I'm using them to provide context for the folks who are going to read it in spanish, while they use github in spanish.

Reply:
Personally, I disagree with the idea that it will be associated with taxes since it's talking about the project's documentation and their users and contributors...
Also in the documentation there's a slight distinction between collaborators and contributors, so with this in mind + the reference from github in spanish I would stick to "contribuyentes"

Reply:
As this is already done, happy to move forward with contribuyente. As a suggestion to avoid having never-ending discussions, let's move the style guide and vocabulary definitions to Slack or other Issue thread and keep evolving this one. The right word may be one of these never-ending discussions, even more if we come from different countries, although this is a very interesting one I have to say.

Reply:
Thanks @flyagaricdev for your contribution and @Stef432 for your comments!